seeds/seed.js

```diff
@@ -0,0 +1,32 @@
+const newFavorites = [
+  {name: 'song1', artist_name: 'artist1', genre: 'genre1', rating: 1},
+  {name: 'song2', artist_name: 'artist2', genre: 'genre2', rating: 2},
+  {name: 'song3', artist_name: 'artist3', genre: 'genre3', rating: 3}
+]
+
+const newPlaylists = [
+    {title: 'playlist1'},
+    {title: 'playlist2'},
+    {title: 'playlist3'}
+]
+
+exports.seed = (knex, Promise) => {
+  return knex('playlists_favorites').del()
+  .then(() => knex('playlists').del())
+  .then(() => knex('favorites').del())
+  .then(() => {
+    return Promise.all([
+      knex('favorites').insert(newFavorites,'id')
+      .then((favorites) => {
+        return knex('playlists').insert(newPlaylists, 'id')
+        .then((playlists) => {
+          return knex('playlists_favorites').insert([
```

Comment:
I'm not entirely sure what the `return` is doing here. Does it break when you remove it?

Reply:
From my understanding, the return is required when you're chaining Promise's together.